lib/pacemaker/pcmk_sched_utilization.c

```diff
@@ -403,7 +403,7 @@ new_load_stopped_op(pcmk_node_t *node)
     char *load_stopped_task = crm_strdup_printf(PCMK_ACTION_LOAD_STOPPED "_%s",
                                                 node->private->name);
     pcmk_action_t *load_stopped = get_pseudo_op(load_stopped_task,
-                                              node->details->data_set);
+                                              node->private->scheduler);
```

Comment:
indent if you need to change anything else, otherwise leave it alone

Reply:
might as well

Reply:
updated commit (only change in push)